Adding that comma seperated lists of indices are supported. (#8040)
opensearch-project/documentation-website: _api-reference/cluster-api/cluster-health.md

GET _cluster/health/<index>
```

> ## URL Parameters

Reviewer: ```suggestion
## Path parameters

```

> The following table lists the available URL parameters. All URL parameters are optional.

Reviewer: ```suggestion
The following table lists the available path parameters. All path parameters are optional.
```

> Parameter | Type | Description

Reviewer: ```suggestion
Parameter | Data type | Description
```

:--- | :--- | :---

> &lt;index-name&gt; | String | Limit health reporting to a specific index. Can be a comma-separated list of multiple index names.

Reviewer: ```suggestion
&lt;index-name&gt; | String | Limits health reporting to a specific index. Can be a single index or a comma-separated list of index names.
```
